scripts/database_helper.py

```diff
@@ -0,0 +1,20 @@
+import psycopg2
+import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+def connect_to_db(db_url):
+    print('Connecting to the PostgreSQL database...')
+    engine = create_engine(db_url, echo=False)
+    Session = sessionmaker(bind=engine)
+    return Session()
+
+def denormalised_reviews_to_df(db_url):
+    engine = create_engine(db_url, echo=False)
+    query_str = 'SELECT A.name, A.category, A.latitude, A.longitude, A."placeId", A.address, A.city, A.locality, B.id, B.ip, B."availabilityInfo", B."safetyInfo", B."updatedAt", B."openingTime", B."closingTime", B.flag, B.deleted FROM "StoreInfos" as A inner join "StoreUpdates" as B on A.id = B."storeId" where B.deleted = False;'
```

Comment:
`A.latitude, A.longitude`-- at this stage, we have 3 overlapping fields: latitude, longitude and coordinate. lat/lng are remnants from google sheet, and coordinate is required for postgis queries. i think we can move to coordinate only in a bit and clean this up. would that work for your scripts?

Reply:
cc @siddddhant in case he has an opinion on this

Reply:
It works either way, this is to flag stores that do not have any location info.